LICENSE

```diff
@@ -36,6 +36,7 @@ The following files and directories are exceptions:
 * vendor/assets/javascripts/jquery.jeditable.js is MIT licensed
 * vendor/assets/javascripts/jquery.jnotify.js is ASL 2.0 licensed
 * vendor/assets/javascripts/jstz.js is MIT licensed
+* vendor/assets/javascripts/ipaddr.js is MIT licensed
```

Comment:
It allows us to use bundler, notice version changes, upgrade just the dependency... In this case, but if a gemified asset dependency has a vulnerability, you can just update that package and it's sorted. When the asset itself is vendored in Foreman you can't do that.